docs/datasets_and_tables/transitdatabase.md

```diff
@@ -2,14 +2,52 @@
 
 ## Data
 
```

Comment:
Can you please add an overall description of the transit database? It would be helpful to know its purpose and what data should live in it.

Reply:
Added in https://github.com/cal-itp/data-infra/pull/802/commits/08a0e42dc20a7b86689f9cbde1cb33efe88e761f. LMK what you think!